File: Bindings~/java/src/main/java/ro/altom/alttester/Notifications/AltLoadSceneNotificationResultParams.java
package ro.altom.alttester.Notifications;

public class AltLoadSceneNotificationResultParams {
    public String sceneName;
    public LoadSceneMode loadSceneMode;

    public AltLoadSceneNotificationResultParams(String sceneName, LoadSceneMode loadSceneMode) {
        this.sceneName = sceneName;
        this.loadSceneMode = loadSceneMode;
    }
}

File: Assets/AltTester/Runtime/AltDriver/Notifications/AltLoadSceneNotificationResultParams.cs
/*
    Copyright(C) 2026 Altom Consulting
*/


namespace AltTester.AltTesterSDK.Driver.Notifications
{
    public class AltLoadSceneNotificationResultParams
    {
        public string sceneName;
        public AltLoadSceneMode loadSceneMode;

        public AltLoadSceneNotificationResultParams(string sceneName, AltLoadSceneMode loadSceneMode)
        {
            this.sceneName = sceneName;
            this.loadSceneMode = loadSceneMode;
        }
    }
}
